src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100000.mdx

```diff
@@ -0,0 +1,76 @@
+---
+subject: 'Python agent'
+releaseDate: '2024-08-08'
+version: 10.0.0
+downloadLink: 'https://pypi.python.org/pypi/newrelic'
+features: ['Remove Python2.7 support', 'Add Kafka server metric', 'Add capturing of memcache host and port', 'Add support for kafka-python-ng', 'Add support for uvicorn_workers', 'Automatically detect function signature in LangChasimilarity search', 'Add environment variable for GC runtime metrics']
+bugs: ['Fix bug in graphql entity name detection', 'Fix issue in LangChain where a thread is started and the transaction context is lost', 'Fix parsing of request and response when running OpenAI via LangChain', 'Instrument new path to OpenAI chat class', 'Fix pyscopg v3 API incompatibilities']
+security: []
+---
+
+## Notes
+
+This release of the Python agent drops support for Python2.7, adds a Kafka server metric, capturing of memcache host and port, support for [kafka-python-ng](https://pypi.org/project/kafka-python-ng/), support for [uvicorn_worker](https://pypi.org/project/uvicorn-worker/), an environment variable for GC runtime metrics, and automatically detects the function signature in [LangChain](https://pypi.org/project/langchain/) `similarity_search`. It also fixes a bug in [graphql](https://pypi.org/project/graphql/) entity name detection, an issue in [LangChain](https://pypi.org/project/langchain/) where a thread is started and the transaction context is lost, parsing of the request and response when running [OpenAI](https://pypi.org/project/openai/) via [LangChain](https://pypi.org/project/langchain/), [pyscopg](https://pypi.org/project/psycopg/) v3 API incompatibilities, and instruments a new path to the [OpenAI](https://pypi.org/project/openai/) chat class.
```

Comment:
I think this is a big enough list that we should use bullet points.  Something like this:

This release of the Python agent drops support for Python2.7 and adds the following:

- a Kafka server metric
- `host` and `port` attributes for memcache
- support for [kafka-python-ng](https://pypi.org/project/kafka-python-ng/)
- support for [uvicorn_worker](https://pypi.org/project/uvicorn-worker/)
- an environment variable for garbage collector (GC) runtime metrics
- automatic detection for the function signature in [LangChain](https://pypi.org/project/langchain/)'s `similarity_search`
- updated support for [OpenAI](https://pypi.org/project/openai/)'s chat class
- a fix for a bug in [graphql](https://pypi.org/project/graphql/)'s entity name detection
- a fix for transaction context propagation loss in [LangChain](https://pypi.org/project/langchain/) in the case of a new thread
- parsing of the request and response when running [OpenAI](https://pypi.org/project/openai/) via [LangChain](https://pypi.org/project/langchain/)
- a fix for [pyscopg](https://pypi.org/project/psycopg/) v3 API incompatibilities